lusidtools/cocoon/cocoon_printer.py

```diff
@@ -0,0 +1,159 @@
+import pandas as pd
+import numpy as np
+
+
+def check_errors_and_success(target_dict: dict, target_name: str, required_keys: list):
+    """
+    This function checks that a named dictionary contains a list of required keys
+    :param dict target_dict: Dictionary to check for keys in
+    :param str target_name: Variable name of dictionary
+    :param list required_keys: List of required keys
+    :return:
+    """
+    for key in required_keys:
+        if key not in target_dict.keys():
+            raise ValueError(f"'{key}' missing from {target_name}. {target_name} must include 'errors', and 'success' "
+                             f"as keys")
+
+
+def get_errors_from_response(list_of_API_exceptions):
+    """
+    This function gets the status code and reason from API exception
+    :param list_of_API_exceptions:
+    :return:
+    """
+    # get API exception
+
+    status = [item.status for item in list_of_API_exceptions]
+    reason = [item.reason for item in list_of_API_exceptions]
+    items_error = np.array([reason, status]).T.tolist()
+    errors = pd.DataFrame(items_error, columns=["error_items", "status"])
+
+    return errors
```

Comment:
Nice! Can you please annotate this with a type i.e. 

def get_errors_from_response(list_of_API_exceptions: list):
    """
    This function gets the status code and reason from API exception
    :param list list_of_API_exceptions:
    :return:
    """

Can you also please add the return type, name and details.